tests/NHSD.BuyingCatalogue.Testing.Data/Entities/EntityBase.cs

```diff
@@ -19,5 +19,10 @@ protected EntityBase()
 
         public async Task InsertAsync()
             => await SqlRunner.ExecuteAsync(ConnectionStrings.GPitFuturesSetup, InsertSql, this).ConfigureAwait(false);
```

Comment:
Probably worth clearing the `ConfigureAwait` while here.

Reply:
👍